### ISSUE TITLE
Add the Other ID Number to the Agency navigator

### ISSUE THREAD

**Issue by fishbaugher** [fishbaugher/WAweb-Issues#33]
It would be very nice to have another column to sort and search in the Agency form.  The Other ID Number could be added as a narrow column to the navigator pane at the bottom of the form.  In MN, this ID number is used to record the service provider number (SP number) and it is often desirable to sort the agency list by this index.

**Comment by fishbaugher** [comment]
Work around it to break first normal database form and start encoding things in comment fields.

**Comment by eckmanwe** [comment]
As you have noted, there is already a workaround (and one that I've seen other Grantees use). 

So is the recommendation to add "Other ID Number" as a possible column in the All Agencies table?

**Comment by fishbaugher** [comment]
Yes, if the Other ID is added to the navigator, it can be used to filter and sort the list which would be useful.

Perhaps replace the "State" field which for a state database does not contain much that is useful for navigation, filtering, or sorting.

Also, this is useful only for site or state level logons which are the only privilege levels that show all agency records.
